pkg/trace/operation.go

```diff
@@ -169,7 +177,7 @@ func FromContext(ctx context.Context) (Operation, error) {
 	case operation:
 		o.operation = val
 	default:
-		return Operation{}, fmt.Errorf("not an Operation")
+		return Operation{Context: context.Background()}, fmt.Errorf("not an Operation")
```

Comment:
@matthewavery Context is not a part of data, that is why it is complaining. Context is a thing that data belongs to. Attaching Context to the data leads to complicated structure re-usage later. It doesn't actually mean, it is the only truth here, there are different possible cases for storing context which are very rare.